libmachine/provision/ubuntu_systemd_test.go

```diff
@@ -27,3 +32,34 @@ func TestUbuntuSystemdCompatibleWithHost(t *testing.T) {
 	}
 
 }
+
+func TestUbuntuSystemdStorageDriver(t *testing.T) {
+
+	p := NewUbuntuSystemdProvisioner(&fakedriver.Driver{}).(*UbuntuSystemdProvisioner)
```

Comment:
Can you please convert this and the other test to use a [test table](https://github.com/golang/go/wiki/TableDrivenTests)?  It seems to be a good fit for what you're doing here, and there are a couple other examples of doing this across the codebase (e.g. `commands/env_test.go`) if you want to take a look.